services/core-api/app/api/mines/permits/permit_conditions/resources/standard_permit_conditions_resource.py

```diff
@@ -0,0 +1,95 @@
+from flask_restplus import Resource, marshal
+from flask import request, current_app
+from werkzeug.exceptions import BadRequest
+from marshmallow.exceptions import MarshmallowError
+
+from app.extensions import api, db
+from app.api.mines.response_models import STANDARD_PERMIT_CONDITION_MODEL
+from app.api.mines.permits.permit_conditions.models import StandardPermitConditions
+from app.api.utils.access_decorators import requires_role_edit_standard_permit_conditions
+from app.api.utils.resources_mixins import UserMixin
+
+    
+class StandardPermitConditionsResource(Resource, UserMixin):
+    @api.doc(description='Update a standard permit condition')
+    @requires_role_edit_standard_permit_conditions
+    @api.expect(STANDARD_PERMIT_CONDITION_MODEL)
+    @api.marshal_with(STANDARD_PERMIT_CONDITION_MODEL, code=200)
+    def put(self, standard_permit_condition_guid):
+        old_display_order = None
+        old_condition = StandardPermitConditions.find_by_standard_permit_condition_guid(
+            standard_permit_condition_guid)
+
+        if old_condition:
+            old_display_order = old_condition.display_order
+
+        try:
+            condition = StandardPermitConditions._schema().load(
+                request.json,
+                instance=StandardPermitConditions.find_by_standard_permit_condition_guid(standard_permit_condition_guid))
+        except MarshmallowError as e:
+            raise BadRequest(e)
+
+        conditions=[]
```

Comment:
I think the formatter needs to run in this file. There should be spaces around the equals sign. I recommend enabling formatting on save.